generic/tests/cfg/unattended_install.cfg

```diff
@@ -37,6 +37,16 @@
     # Fedora guests. For other guests like ubuntu if your install failed with
     # don't have enough RAM error from anaconda, please enlarge this value.
     lowest_mem = 512
+    variants:
+        - @default:
+        - multivm:
+            unattended_install_multivm = "yes"
+            thread_timeout = "3600"
+            variants:
+                - orderly:
+                    unattended_install_type = "orderly"
+                - simultaneous:
+                    unattended_install_type = "simultaneous"
```

Comment:
Instead of using variants (`@default` vs `multivm`), we can make it as the default for multiple VMs test, how would you think about that?

Reply:
you mean to make `multivm` test as default ? It will be great, I can make change if you are okay.

Reply:
I mean the "default" is depended on how user sets the `vms` param: if there is only one VM (e.g. `vms = vm1`), that means user wants the `default` variant; otherwise, user wants the other (`multivm`). So, we don't have to introduce the additional variants (`@default` vs `multivm`).

Reply:
okay, I will make the change as suggested.

Reply:
@luckyh I have made the changes as suggested by you, also removed `orderly` variant. please help to review. I have given new param to handle orderly or random installation based on user input.